tests/logic/email_test.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+import mock
+import unittest
+
+import logic.email
+
+
+class EmailTest(unittest.TestCase):
+    """We really just want to test that configuration is honored here."""
+
+    sender = 'test@example.com'
```

Comment:
When sender or recipient gets changed to **Test <test@example.com>** the `get_name_and_email` function would be completely tested. Right now it‘s just partially tested.